src/store/flow/block-types/MobilePrimitives_NumericResponseBlockStore.ts

```diff
@@ -13,50 +13,52 @@ export const getters: GetterTree<IFlowsState, IRootState> = {}
 export const mutations: MutationTree<IFlowsState> = {}
 
 export const actions: ActionTree<IFlowsState, IRootState> = {
-  async setValidationMinimum({commit, rootGetters}, {blockId, value} : {blockId: string, value: number}) {
+  async setValidationMinimum({ commit, rootGetters }, { blockId, value }: {blockId: string; value: number}) {
```

Comment:
Hi @poojahpatil90, have we matched our config to that of flow runner? 

I thought that project allowed us to skip braces spaces on object literals and destructured assignments (+ imports) 🤷‍♀️

Reply:
@bzabos we have not matched our config with flow runner, we can do so with VMO-3551. I will add it as part of the description to keep track of it.